app/controllers/application_controller.rb

```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
+  helper ApplicationHelper
```

Comment:
I'm surprised that this was necessary.  I thought rails did this for us automagically.